extension/ballastextension/testdata/config.yaml

```diff
@@ -0,0 +1,20 @@
+extensions:
+  memory_ballast:
+  memory_ballast/1:
+    ballast_size_mib: 123
```

Comment:
Could be just `size_mib` as extension name makes the rest obvious.

Reply:
nice, will do that.

Reply:
Done.